tests/gnmi/helper.py

```diff
@@ -234,6 +234,124 @@ def gnmi_get(duthost, ptfhost, path_list):
             raise Exception("error:" + msg)
 
 
+# py_gnmicli does not fully support POLLING mode
+# Use gnmi_cli instead
+def gnmi_subscribe_polling(duthost, ptfhost, path_list, interval_ms, count):
+    """
+    Send GNMI subscribe request with GNMI client
+
+    Args:
+        duthost: fixture for duthost
+        ptfhost: fixture for ptfhost
+        path_list: list for get path
+        interval_ms: interval, unit is ms
+        count: update count
+
+    Returns:
+        msg: gnmi client output
+    """
+    env = GNMIEnvironment(duthost, GNMIEnvironment.GNMI_MODE)
+    ip = duthost.mgmt_ip
+    port = env.gnmi_port
+    interval = interval_ms / 1000
```

Comment:
>/ [](http://example.com/codeflow?start=27&length=1)

Is your intention integer division or float division?

Reply:
Fixed.